astroquery/alma/core.py

```diff
@@ -1105,3 +1063,27 @@ def unique(seq):
 def filter_printable(s):
     """ extract printable characters from a string """
     return filter(lambda x: x in string.printable, s)
+
+
+def uid_json_to_table(jdata,
+                      productlist=['ASDM', 'PIPELINE_PRODUCT',
+                                   'PIPELINE_PRODUCT_TARFILE',
+                                   'PIPELINE_AUXILIARY_TARFILE']):
+    rows = []
+
+    def flatten_jdata(jj):
```

Comment:
nitpicking but would be nice to use something meaningful rather than `jj` and `kk`. Arguably these are still way much better than single letter vars 😅

Reply:
They are much better!  You can `s/jj/blahblah/` but you cannot `s/j/blah/`

Reply:
yes, I acknowledged that 😜

Reply:
You said "arguably".  I had to argue.  QED.